fr.janssoone.pkgtransporter.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "fr.janssoone.pkgtransporter",
```

Comment:
is `janssoone.fr` really controlled by the upstream developer ?

Reply:
Non this romain don't exist, I've misunderstood.
Could replace it with com.gitlab ?

Reply:
see
https://docs.flatpak.org/en/latest/conventions.html#application-ids

linked from
https://docs.flathub.org/docs/for-app-authors/requirements#application-id

Reply:
Thanks,
I've changed the appId to `io.gitlab.gwendalj.package-transporter`.

Reply:
Not yet, I was waiting the answer for the other comment before pushing this change